### PR TITLE
Fixes #8003: update list of technique in maintained technique

### PR DESCRIPTION
**Pull request by ncharles**
https://www.rudder-project.org/redmine/issues/8003

### CODE REVIEW

**Comment by ncharles** [comment]
PR updated

File: maintained-techniques [hunk 1]
```diff
@@ -72,6 +73,7 @@ systemSettings/misc/clockConfiguration/1.0
 systemSettings/misc/clockConfiguration/2.0
 systemSettings/misc/clockConfiguration/3.0
 systemSettings/misc/clockConfiguration/3.1
+systemSettings/misc/clockConfiguration/4.0
```
Comment (jooooooon) [inline maintained-techniques]: This version is not present in the 2.11 branch:

```
$ git status
On branch branches/rudder/2.11
Your branch is up-to-date with 'origin/branches/rudder/2.11'.

nothing added to commit but untracked files present (use "git add" to track)
$ ls systemSettings/misc/clockConfiguration
1.0  2.0  3.0  3.1
```

Comment (ncharles) [inline maintained-techniques]: ha - my repo has an issue then ...

**Comment by ncharles** [comment]
PR updated

**Pull request merged**